irohad/ametsuchi/impl/storage_impl.hpp

```diff
@@ -15,30 +15,21 @@
  * limitations under the License.
  */
 
-#ifndef IROHA_COMMANDEXECUTOR_HPP
-#define IROHA_COMMANDEXECUTOR_HPP
+#ifndef IROHA_STORAGE_IMPL_HPP
+#define IROHA_STORAGE_IMPL_HPP
 
-#include <model/command.hpp>
+#include <ametsuchi/storage.hpp>
 
 namespace iroha {
-
   namespace ametsuchi {
-    /**
-     * Applies command to the world state view
-     */
-    class CommandExecutor {
+    class StorageImpl : public Storage {
```

Comment:
Why is it inheritance?
I think `Storage` is right.
Because implementation of `Storage` is always not changing if the database is changed.

Maybe, It is for test?